csharp/test/Ice/binding/AllTests.cs

```diff
@@ -483,7 +483,7 @@ public static void Run(TestHelper helper)
 
                 // Test IPv6 dual mode socket
                 {
-                    using Communicator serverCommunicator = new Communicator();
```

Comment:
You should change the local variables in this file to use `var` instead of `new()` like you did the other ones.